src/lammpsio/snapshot.py

```diff
@@ -352,17 +409,51 @@ def to_hoomd_gsd(self, type_map=None):
 
     @property
     def N(self):
-        """int: Number of particles."""
+        """int: Number of particles.
+
+        Example
+        -------
+        .. code-block:: python
+
+            num_particles = snapshot.N
```

Comment:
I found these little snippet examples very unhelpful, and they're not used consistently in all classes. I would suggest to remove unless you have a strong preference to keep them, or you want to spend time to make them more meaningful.